dm/dm-table-routing.md

```diff
@@ -184,7 +184,7 @@ mysql> select * from test.t;
 ```sql
 CREATE TABLE `test`.`t` (
     c_table varchar(10) DEFAULT NULL,
-    a int(11) PRIMARY KEY,
+    a int PRIMARY KEY,
```

Comment:
Same here, where does it origin from and will it be with or without display width?